### PR TITLE
Align with Atk 2.4

### PR DESCRIPTION
**Pull request by abbadon1334**
Align to Atk4/Ui 2.4

Add an helper class for download and view of stored files.

### CODE REVIEW

File: src/Helper.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            'Content-Disposition' => 'attachment; filename="' . $model->get('meta_filename') . '"',
+            'Content-Length' => (string) $model->get('meta_size'),
+            'Pragma' => 'public',
+            'Accept-Ranges' => 'bytes',
+        ];
+
+        static::output($model, $headers, $app);
+    }
+
+    private static function output(File $model, array $headers, App $app)
```
Comment (DarkSide666) [inline src/Helper.php:28]: `$app` should be defined as `App $app = null` allowing null value.
Also you could set default value for $headers to empty array `array $headers = []`

File: src/Helper.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    {
+        $location = $model->get('location');
+
+        if ($app !== null) {
+            $app->terminate($model->flysystem->get($location)->read(), $headers);
+        }
+
+        foreach ($headers as $k => $v) {
+            header($k . ': ' . $v);
```
Comment (DarkSide666) [inline src/Helper.php:37]: Shouldn't `$v` be escaped in some way here?

**Review by DarkSide666 — APPROVED** [review]
Looks OK, but please see my comments and fix before merge.

File: src/Helper.php (hunk 1 above)
Comment (abbadon1334) [reply src/Helper.php]: my fault, i add it.

File: src/Helper.php (hunk 2 above)
Comment (abbadon1334) [reply src/Helper.php]: don't think so, but I copy normalization from Atk4/Ui to be sure.

**Review by DarkSide666 — APPROVED** [review]

**Pull request merged**